### PR TITLE
update .NET logo

### PR DESCRIPTION
**Pull request by BillWagner**
The official .NET logo should be square. Update source from @BethMassi

### CODE REVIEW

**Comment by BillWagner** [comment]
@BethMassi Can you take a look at the [internal review site](https://review.docs.microsoft.com/en-us/dotnet/?branch=pr-en-us-15536) The updated logo looks like a slightly different shade of purple.

**Comment by BethMassi** [comment]
Yes, that's the official logo.

**Comment by BillWagner** [comment]
@BethMassi Should the others change color? Or is that OK for now?

**Comment by BethMassi** [comment]
I think it's OK for now. I'll talk to DEVDIV UX about it.

**Comment by Thraka** [comment]
Why is the SVG just displaying an image instead of being made up of paths?

**Comment by BethMassi** [comment]
Here's the formats I have. Please pick one to use as this is our official logo and needs to be somewhere on the docs page. 
https://github.com/dotnet/swag/blob/master/netlogo/xsmall-155x155/

File: index.md [hunk 1]
```diff
@@ -82,7 +82,7 @@ ms.date: "09/30/2019"
                                         <div class="card">
                                             <div class="cardImageOuter">
                                                 <div class="cardImage">
-                                                    <img src="/dotnet/images/hub/dotnet-logo-image.svg" alt="Screenshot of the purple .NET logo." />
+                                                    <img src="/dotnet/images/hub/netlogo-purple.svg" alt="Screenshot of the purple .NET logo." />
```
Comment (mairaw) [inline index.md]: can we just replace the existing one? it has better SEO (that's why I ask)

**Review by mairaw — COMMENTED** [review]
It looks a bit strange to have that to be super different than the others. Do we have official logo for the others? Do we need to change all the .NET related ones to be square as well at least?

**Comment by Thraka** [comment]
@BethMassi Can those be exported as SVG? Barely anyone in the docs org would have a license for adobe.

**Comment by BethMassi** [comment]
Isn't that the format @BillWagner used?

**Comment by BillWagner** [comment]
I updated the file name to match the earlier version.

@BethMassi Yes, you gave me an SVG.  The difference is that most of them have text path information. See [this example](https://raw.githubusercontent.com/dotnet/docs/master/docs/images/hub/dotnet-core.svg) The new logo SVG has embedded PNG information. See [here](https://raw.githubusercontent.com/dotnet/docs/030f00a979ed8cf3c880b3fd298b1a02b7199e93/docs/images/hub/dotnet-logo-image.svg)

**Comment by Thraka** [comment]
@BethMassi I also meant that it would be good to have those in SVG format where you linked. If I need to use an icon for something, I know where to go. 😄 But as I and Bill pointed out, the SVG that was given to bill is just wrapping a PNG which means there is a limit to the scaling. Additionally, it makes it hard to customize if needed. For example, I was able to edit the XAML tile here: https://docs.microsoft.com/en-us/dotnet/#pivot=docs&panel=desktop and add the core/framework logos for the distinction.

**Review by Thraka — APPROVED** [review]
The preview site stinks and I can't see what it looks look. But I assume it's all OK 😄

**Pull request merged**